src/test/java/elizCurtnazar/tests/tema8/teste/RefreshPageTests.java

```diff
@@ -0,0 +1,17 @@
+package elizCurtnazar.tests.tema8.teste;
+
+import elizCurtnazar.tests.tema8.pageObject.RefreshPage;
+import org.testng.annotations.Test;
+
+public class RefreshPageTests extends BaseTest {
+
+    @Test(groups = {"mobile","desktop"})
+    public void refreshPage() {
+        driver.get("https://testpages.herokuapp.com/styled/refresh");
+        RefreshPage refreshPage = new RefreshPage(driver);
+        refreshPage.getTimeOfTheRefresh();
+        System.out.println("Am intrat pe pagine de refresh si timpul este " + refreshPage.getTimeOfTheRefresh());
+        driver.navigate().refresh();
+        System.out.println("Am dat un refresh si timpul acum este " + refreshPage.getTimeOfTheRefresh());
```

Comment:
You're missing an assertion here for this scenario. Please add it. Thanks!

Reply:
done